<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-EH37MPF" failures="1" tests="1" name="runner.TestNGRunner" time="10.291" errors="0" timestamp="2024-07-14T09:02:55 IST" skipped="0">
  <testcase classname="runner.TestNGRunner" name="runScenario" time="10.291">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at pages.ProductsPage.prices_after_filter(ProductsPage.java:58)
at stepdefination.SaucedemoStepdef.verfify_prices_after_filter(SaucedemoStepdef.java:58)
at âœ½.verify the Prices of the products after filter(file:///E:/Workspace_2023/SauceLab/src/test/resources/Features/Saucedemo.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runner.TestNGRunner -->
